conf/services/pension-practitioners.yml

```diff
@@ -0,0 +1,29 @@
+serviceName: Managing Pension Schemes
+serviceHeaderName: Managing Pension Schemes
```

Comment:
@Ashwani1403 The parameter `serviceHeaderName` is no longer needed.

Reply:
ok, update done